backend-proxy-remote.conf.js

```diff
@@ -27,7 +32,7 @@ const PROXY_CONFIG = [
             }
         },
         "target": backendhostname,
-        "secure": true,
+        "secure": backendhostsecure,
```

Comment:
We should do the same in the websocket section.

Reply:
Thanks @tadzik; should be fixed now.